public/app/plugins/panel/logs/LogsPanel.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { LogRows, CustomScrollbar } from '@grafana/ui';
+import { css } from 'emotion';
```

Comment:
as per https://emotion.sh/docs/emotion-11#package-renaming
```suggestion
import { css } from '@emotion/css';
```

we don't directly depend on `emotion`, it's installed as a transient dependency of `react-diff-viewer`